### PR TITLE
add-greeting-by-BlueTeam

### PR DESCRIPTION
**Pull request by kolomame**
blue team

### CODE REVIEW

**Comment by 10kinonik01** [comment]
OK

**Review by 10kinonik01 — COMMENTED** [review]
OK

**Review by kolomame — COMMENTED** [review]
add blueTeam

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <div class="container">
         <h1>Hello Recursion</h1>
         <h1>Hello Red Team</h1>
+        <h1>Hello Blue Team</h1>
```
Comment (10kinonik01) [inline index.html:14]: Ok

**Pull request merged**